### PR TITLE
Fix underselection problem in invertlogical refactoring.

### PR DESCRIPTION
**Pull request by petrroll**

### CODE REVIEW

File: src/Features/Core/Portable/InvertLogical/AbstractInvertLogicalCodeRefactoringProvider.cs [hunk 1]
```diff
@@ -57,6 +57,18 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                     expression = expression.Parent;
                 }
             }
+            else
+            {
+                // When selection is non-empty -> allow only top-level full selections.
```
Comment (mavasani) [inline src/Features/Core/Portable/InvertLogical/AbstractInvertLogicalCodeRefactoringProvider.cs]: This seems like a reasonable restriction to me. If we do get feedback that the lost functionality is indeed desired, we can revert this change.

Comment (petrroll) [reply src/Features/Core/Portable/InvertLogical/AbstractInvertLogicalCodeRefactoringProvider.cs]: There's no lost functionality. The refactoring previously didn't allow selections at all. Maybe 16.3 already has enabling selections & won't get this fix but I'm pretty sure that's not the case.
